gufe/__init__.py

```diff
@@ -16,12 +16,6 @@
 
 from .chemicalsystem import ChemicalSystem
 
-from .mapping import (
```

Comment:
We can't move all these imports, this will break the API

Reply:
shame :( I preferred that structure.